docs/compatibility.md

```diff
@@ -25,14 +25,14 @@
 | getShortName | :heavy_check_mark: Yes |
 | getStartLine | todo |
 | getStaticProperties | todo |
-| getStaticPropertyValue | :x: No |
+| getStaticPropertyValue | :x: No - would require an instance |
```

Comment:
You can create an accessor even without the instance of the object or internal reflection.

See http://ocramius.github.io/blog/accessing-private-php-class-members-without-reflection/